packages/google-pay-integration/e2e/GooglePayBraintreeInPaymentStep.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { CustomerStepPreset, test } from '@bigcommerce/checkout/test-framework';
+
+import {
+    afterSignIn,
+    authResponse,
+    beforeSignIn,
+    braintreePayPalCredit,
+    googlePay,
+    internalOrder,
+    order222,
+    orderPayment,
+    payment,
+} from './GooglePayBraintreeMockingResponses';
+
+test.describe('Google Pay Braintree', () => {
+    test('Google Pay Braintree wallet button is working', async ({
+        assertions,
+        checkout,
+        page,
+    }) => {
+        // Testing environment setup
+        let isSignedIn = false;
+
+        const responseProps = { status: 200, contentType: 'application/json' };
+
+        await checkout.use(new CustomerStepPreset());
+
+        await checkout.start('Google Pay Braintree in Customer Step');
+
+        await checkout.route(
+            /order-confirmation.*/,
+            './packages/test-framework/src/support/orderConfirmation.ejs',
+            { orderId: '222' },
+        );
+
+        await checkout.route('**/checkout.php', `${__dirname}/support/checkout.php.ejs`);
+
+        await checkout.route(
+            'https://pay.google.com/gp/p/js/pay.js',
+            `${__dirname}/support/googlePay.mock.js`,
+        );
```

Comment:
`$__dirname/support/` does not appear to exist here.

Reply:
Why? It is exist here

<img width="1404" alt="Screenshot 2023-10-06 at 10 59 12" src="https://github.com/bigcommerce/checkout-js/assets/99336044/e678b264-b734-478f-b2d3-886f563cfdf2">

Reply:
Oh, I see. I didn't notice `$__dirname/support/` was already in the Google Pay package.